homeassistant/components/camera/android_ip_webcam.py

```diff
@@ -0,0 +1,128 @@
+"""
+Support for IP Webcam, an Android app that acts as a full-featured webcam.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.android_ip_webcam/
+"""
+import asyncio
+import logging
+from contextlib import closing
+
+import aiohttp
+import async_timeout
+import requests
+from requests.auth import HTTPBasicAuth
+
+from homeassistant.components.camera import (PLATFORM_SCHEMA, Camera)
+from homeassistant.helpers.aiohttp_client import (
+    async_get_clientsession, async_aiohttp_proxy_stream)
+from homeassistant.components import android_ip_webcam
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({})
+
+DEPENDENCIES = ['android_ip_webcam']
+
+
+@asyncio.coroutine
+# pylint: disable=unused-argument
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Setup an IP Webcam Camera."""
+    if discovery_info is None:
+        return
+    devices = hass.data[android_ip_webcam.DATA_IP_WEBCAM]
+    cameras = [IPWebcamCamera(hass, device)
+               for key, device in devices.items()]
+    yield from async_add_devices(cameras, True)
+
+
+def extract_image_from_mjpeg(stream):
```

Comment:
This was only necessary for the `camera_image` function. Removing this because of removal of `camera_image`.